### DISCUSSION TITLE
How would I go about adding items into the backpack of some jobs?

### DISCUSSION THREAD

**Discussion by Starbuckss14** [space-wizards/space-station-14#32504] [Questions]
I want to add a stack of steel to engineering backpacks on round start. How would I do this?

**Comment by lzk228** [comment]
all special roundstart items for roles (except loadouts) are located in JobPrototype in https://github.com/space-wizards/space-station-14/tree/master/Resources/Prototypes/Roles/Jobs
there are comments for example of adding stuff
